### ISSUE TITLE
Collect expertise areas from editorial board members

### ISSUE THREAD

**Issue by tkuhn** [data-science-hub/data-science-hub.github.io#22]
We need some structured way of collecting the expertise areas of the editorial board members, in support paper assignments and discussions. Maybe this can be done in the submission system.

**Comment by micheldumontier** [comment]
we should use vivo. https://wiki.duraspace.org/display/VTDA/Activating+the+ORCID+integration

**Comment by tkuhn** [comment]
How can we use VIVO for that? Can scientists enter their experise areas in VIVO and we can get them via ORCID integration?

**Comment by micheldumontier** [comment]
That's correct.  However, it's not clear to me that anybody can create a vivo profile without the organization having the server? not sure...

**Comment by tkuhn** [comment]
That seems to be the case. At least, I couldn't figure out how to create a VIVO profile for myself...

**Comment by micheldumontier** [comment]
ok Melissa indicated that Openvivo.org can be used

**Comment by mellybelly** [comment]
this is what we used for the Force2016 conference, openVivo.org can be used by anyone with an ORCID.
In addition to the usual suspects, it also pulls non traditional scholarly contributions from Figshare. Working on other sources too.
